### PR TITLE
Separate site validation and responsibilities into their own pages (SOFTWARE-4266)

### PR DESCRIPTION
**Pull request by brianhlin**
@jdost321 @lmichael107 Check out the sort of information that we ask from sites if there are any changes here: https://github.com/opensciencegrid/docs/compare/master...brianhlin:SOFTWARE-4266.separate-validation-responsibilities-pages?expand=1#diff-4433075fa7de1e0340f2ff184d4cd452

I'll add a section to the Hosted CE doc that points users to this new page and tell them to just contact us if there are any changes.

### CODE REVIEW

File: docs/site-verification.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+=================
+
+After installing and [registering](common/registration.md) services from the [site planning document](site-planning.md),
+you will need to perform some verification steps before your site can
+[scale up to full production](#scale-up-to-full-production).
+
+Verify OSG Software
+-------------------
+
+To verify your site's installation of OSG Software, submit local test jobs, contact the OSG for end-to-end
```
Comment (matyasselmeci) [inline docs/site-verification.md:11]: Could be a bulleted list...

File: docs/site-verification.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+### Local verification ###
+
+It is useful to submit jobs from within your site to verify CE's ability to submit jobs to your local batch system.
+Consult the document for [submitting jobs into an HTCondor-CE](compute-element/submit-htcondor-ce) for detailed
+instructions on how to test job submission.
+
+### Verify end-to-end pilot job submission ####
+
+Once you have validated job submission from within your site, request test pilot jobs from
+[OSG Operations](mailto:osg-gfactory-support@physics.ucsd.edu) and provide the following information:
```
Comment (matyasselmeci) [inline docs/site-verification.md:23]:
```suggestion
[OSG Factory Operations](mailto:osg-gfactory-support@physics.ucsd.edu) and provide the following information:
```
I don't think we want people sending general operational questions there...

File: docs/site-verification.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+[OSG's Accounting Portal](https://gracc.opensciencegrid.org/dashboard/db/site-summary) and select your registered OSG
+site name from the `Site` dropdown.
+If you don't see your site in the dropdown,
+[please contact us for assistance](common/help.md#software-or-service-support).
+
+Scale Up to Full Production
+---------------------------
+
+After verifying end-to-end pilot job submission and usage reporting, your site is ready for production!
+In the same OSG Operations ticket that you opened [above](#verify-end-to-end-job-submission),
```
Comment (matyasselmeci) [inline docs/site-verification.md:49]:
```suggestion
In the same OSG Factory Operations ticket that you opened [above](#verify-end-to-end-job-submission),
```

File: docs/site-verification.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+successfully, that number will be ramped up.
+
+### Verify reporting and monitoring ###
+
+To verify that your site is correctly reporting to the OSG, visit
+[OSG's Accounting Portal](https://gracc.opensciencegrid.org/dashboard/db/site-summary) and select your registered OSG
+site name from the `Site` dropdown.
+If you don't see your site in the dropdown,
+[please contact us for assistance](common/help.md#software-or-service-support).
+
```
Comment (matyasselmeci) [inline docs/site-verification.md:44]: They don't need to check flock.opensciencegrid.org any more?

**Review by matyasselmeci — APPROVED** [review]
A few tweaks but otherwise no biggie. Pre-approving.

File: docs/site-verification.md (hunk 4 above)
Comment (brianhlin) [reply docs/site-verification.md:44]: Good catch but I did exclude that on purpose! Finding your site in the collection of images really isn't user friendly so I dropped it.

**Comment by brianhlin** [comment]
I  believe the linkchecker is failing erroneously due to these new docs. Let's merge and find out!

**Comment by brianhlin** [comment]
It worked!

**Pull request merged**